spec/unit/info_service_spec.rb

```diff
@@ -36,18 +36,18 @@
       it "returns task data for valid tasks in an environment even if invalid tasks exist" do
         Puppet.override(:environments => env_loader) do
           @mod = PuppetSpec::Modules.create(mod_name, modpath, {:environment => env,
-                                                                :tasks => [['atask',
-                                                                            {:name => 'atask.json',
-                                                                             :content => metadata.to_json}],
-                                                                           ['btask',
-                                                                            {:name => 'btask.json',
-                                                                             :content => metadata.to_json}],
-                                                                           ['ctask',
-                                                                            {:name => 'ctask.json',
-                                                                             :content => metadata.to_json}]]})
+                                                              :tasks => [['atask',
+                                                                          {:name => 'atask.json',
+                                                                           :content => metadata.to_json}],
+                                                                         ['btask',
+                                                                          {:name => 'btask.json',
+                                                                           :content => metadata.to_json}],
+                                                                         ['ctask',
+                                                                          {:name => 'ctask.json',
+                                                                           :content => metadata.to_json}]]})
           File.write("#{modpath}/#{mod_name}/tasks/atask.json", "NOT JSON")
 
-          expect(Puppet).to receive(:send_log).with(:err, /unexpected token at 'NOT JSON'/)
+          expect(Puppet).to receive(:send_log).with(:err, /unexpected token|unexpected character|parse error/i)
```

Comment:
I noticed this when I took a pass at the tests earlier (never finished). I wonder why it's different. 🤔